source/usage-examples/replaceOne.txt

```diff
@@ -0,0 +1,46 @@
+==================
+Replace a Document
+==================
+
+.. default-domain:: mongodb
+
+You can replace a single document using the `replaceOne()
+<https://mongodb.github.io/node-mongodb-native/3.3/api/Collection.html#replaceOne>`_
+method. ``replaceOne()`` accepts a query document and a replacement
+document, and replaces the first document that matches the filter with
+the provided replacement document.
```

Comment:
My previous comment may not have been the greatest suggestion. If you use "replaces with", the description probably requires some clarification on whether it replaces the entire document or only the matching fields of the replacement document. Personally, without testing it, I'm not sure which one it is!

Reply:
Added quite a bit of extra information about this -- you're totally right, there's a lot of vagueness here right now. Let me know what you think of my changes.